### ISSUE TITLE
execution plan does not fit in some cases...

### ISSUE THREAD

**Issue by mzatko** [jcgay/buildplan-maven-plugin#5]
As example
```xml
<plugin>
    <groupId>com.comphenix.maven</groupId>
    <artifactId>yaml-properties-plugin</artifactId>
    <version>0.1.0</version>
    <executions>
        <execution>
            <phase>initialize</phase>
            <goals>
                <goal>read-yaml-properties</goal>
            </goals>
            <configuration>
                <files>
                    <!-- YAML files that will be loaded. The element name is the file ID. -->
                    <profile>${project.build.directory}/application-profile.yml</profile>
                    <common>${basedir}/src/main/resources/config/application.yml</common>
                </files>
            </configuration>
        </execution>
    </executions>
</plugin>
```

```
[INFO] yaml-properties-plugin   | validate               | default                  | read-yaml-properties 
```

phase:`validate` in buildplan, phase:`initialize` in pom.xml

**Comment by jcgay** [comment]
Thanks for the report, I will look into it !

**Comment by jcgay** [comment]
This should be fixed in v1.2 released yesterday !

Thanks again.
